### ISSUE TITLE
[Eradicator] awe32/64 "hi-res sound"?

### ISSUE THREAD

**Issue by nemo93** [dosbox-staging/dosbox-staging#1223]
While testing I've stumbled upon a weird option in **Eradicator**. I'm cc'ing below an excerpt from `AWE3264.TXT` found in the game directory.

```
In the sound options menu for Eradicator there is a menu item labeled
AWE32/64 HI-RES SOUND.  If this feature is enabled and your system has one of
the cards mentioned above Eradicator will take advantage of this hardware to
enhance the audio quality of Eradicator.
[...]
Eradicator uses one of two sound font banks to load sound effects for the game into
the RAM on the above mentioned cards. There is a 512K bank named ERAD512K.SF2
and a 2 Meg. bank named ERAD2MEG.SF2. Eradicator will automatically load one 
of these two banks into your sound card based on amount of sound card RAM 
available.
```

Game has redbook audio hence the aforementioned improvement would be for SFX only. 
1. Is this somehow marketing BS or is there really any benefit in using such soundcards?
2. Are the AWE32/64 cards emulated in DOSBox/Staging?

**Comment by kklobe** [comment]
I don't see it any references in the code after a quick scan, and, from the README:

```
- SoundBlaster 16 / SoundBlaster Pro I & II / SoundBlaster I & II
      By default DOSBox provides SoundBlaster 16 level 16-bit stereo sound.
      You can select a different SoundBlaster version in the configuration of
      DOSBox. AWE32 music is not emulated as you can use MPU-401 instead
      (see below).
```

**Comment by nemo93** [comment]
Thanks @kklobe for checking and apologize for the time waste.

**Comment by johnnovak** [comment]
Support for Eradicator will be added as part of this ticket:
https://github.com/dosbox-staging/dosbox-staging/issues/1682

**Comment by Burrito78** [comment]
This will be implemented and checked if its working in #1682 we don't need to have two tickets for this i think.
